test/language/expressions/arrow-function/arrow/CoverParenthesizedExpressionAndArrowParameterList-expression-3.js

```diff
@@ -0,0 +1,9 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: "CoverParenthesizedExpressionAndArrowParameterList :( Expression )"
+es6id: 14.2
+---*/
+
+assert.sameValue(((a, b) => this)(), this, "CoverParenthesizedExpressionAndArrowParameterList :( Expression ) failed for (a, b) => this.");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js
- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/syntax/arrowparameters-cover-concisebody-assignmentexpression.js